### DISCUSSION TITLE
Вопросик

### DISCUSSION THREAD

**Discussion by OihhoMoiss** [Flowseal/zapret-discord-youtube#812] [Q&A] [answered]
Здравствуйте, я хотел бы узнать, является ли эта программа VPN-ом?
Я играю в игры, где за использование VPN могут забанить, поэтому я хочу понимать, есть ли риски при использовании «Запрета»

**Comment by Eptiy** [comment, marked as answer]
Это не впн. Колда, например, работать не будет, и, возможно, есть ещё какие-то конфликты с античитами, но бан не прилетит.

**Reply by OihhoMoiss** [reply]
Спасибо за ответ

**Comment by Liber142** [comment]
Ошибка: Запуск программы невозможен, так как на компьютере отсутсвует sygwin1/dll 
Помогите решить. Вроде установил этот файл и куда пихать его непонимаю
